src/worker/index.ts

```diff
@@ -212,10 +221,6 @@ export default class Worker {
         }
 
         // Index new pending transactions
-        const newPendingTransactions = _.filter(
-            transactions,
-            pending => !_.includes(indexedHashes, pending.hash().value)
-        );
-        await TxModel.createTransactions(newPendingTransactions, true);
+        await TxModel.createTransactions(transactions, true);
```

Comment:
Is it okay? My concern is, it seems that `createTransactions()` can fail due to a hash conflict.